packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -83,7 +82,6 @@ const AddOrManageAccounts = props => {
                 createCryptoAccount(value)
                     .then(() => {
                         onSuccessSetAccountCurrency('', value, deposit_target);
-                        setSubmitting(false);
```

Comment:
here we were calling this formik function after the component was replaced with another component/screen in the modal. so this `setSubmitting(false); `is not required for an unmounted form.